packages/experiments/src/components/Form/inputs/textInput/FormTextInput.test.tsx

```diff
@@ -108,8 +107,10 @@ describe('FormTextInput Unit Tests', () => {
       ReactTestUtils.Simulate.submit(form);
 
       // Find the TextField component
-      const field = ReactTestUtils.findRenderedComponentWithType(renderedForm, TextField);
-      expect(field.state.errorMessage).toBeTruthy();
+      // TODO: this test is not working as intended as even nonsense state names pass.
+      //        commented out for now since test isn't executing when named 'xit'
+      // const field = ReactTestUtils.findRenderedComponentWithType(renderedForm, TextField);
```

Comment:
ive seen these tests in form tests randomly fail. We will need to evaluate what the right pattern is here once we gain traction.